runtime/browser/xwalk_pref_store.cc

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2013 The Chromium Authors. All rights reserved.
```

Comment:
I think there is code change beside copy from chromium, if yes, please add Intel copyright